webapp/javascript/components/Settings/APIKeys/APIKeyAddForm.tsx

```diff
@@ -101,7 +103,25 @@ function APIKeyAddForm() {
               required
             />
             <div>
-              <h4>Role</h4>
+              <h4>
+                Role
+                <span
+                  className={styles.infoMark}
+                  onMouseEnter={() => setRolesTooltipVisibility(true)}
+                  onMouseLeave={() => setRolesTooltipVisibility(false)}
+                >
+                  <Tooltip
+                    title={
+                      'Admin - Manage users/API Key using the HTTP API\n' +
```

Comment:
@eh-am not sure about this but it's just managing users / api keys right? Not just via the http api
```suggestion
                      'Admin: Manage users/API keys\n' +
```